specification/containerservice/resource-manager/Microsoft.ContainerService/aks/stable/2024-09-01/managedClusters.json

```
@@ -4982,7 +4982,7 @@
           "additionalProperties": {
             "$ref": "#/definitions/UserAssignedIdentity"
           },
-          "description": "Identities associated with the cluster."
+          "description": "The user identity associated with the managed cluster. This identity will be used in the kubelet. Only one user assigned identity is allowed, pass {\"kubeletIdentity\":{\"resourceId\":\"/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{identityName}\"}}.",
```

Might be clearer for other contexts where raw JSON isn't the input to phrase it something like:
```suggestion
          "description": "The user identity associated with the managed cluster. This identity will be used by the kubelet. Only one user assigned identity is allowed. Only only accepted key is \"kubeletidentity\", with value of \"resourceId\": \"/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{identityName}\".",
```

---

Done!